### ISSUE TITLE
Cover other legal areas

### ISSUE THREAD

**Issue by dirk-qualisys** [esdscom/sdscom-xml#98]
In the supply chain, many customers ask for additional data to apply a product outside the area of the delivered SDS, E.g., it would be helpful to submit the notification state to national inventories. 

This requires a scope enhancement to other legal regions. Should this be done? And should it cover other SDS formats completely (e.g. US OSHA HazCom 2012)? If so, will this be combined into one XML structure as all formats are derived from GHS, or is the information or presentation so different that we accept the redundancies of separate structures?

**Comment by dirk-qualisys** [comment]
Today's Steering Committee meeting decided to start version 5.0 directly after publishing version 4.4.0, including SDS formats for any regions that is interesting for users/implementers (and where somebody is willing and able to contribute the respective format).

For this, InformationFromExportingSystem will be pulled out of the SDS and appear only once per file. But although data in sections 9, 11, 12 is the same for all national varieties of the SDS, it stays in the Datasheet node (and is not moved in a product level node) to avoid the necessity of providing all translated free texts in one phrase. This improves compatibility with version 4.4.0 and makes implementation easier.

**Comment by dirk-qualisys** [comment]
As a first step, in every stucture (subtree) that contains information specific to the European SDS, this info should be collected in a separate tag "RegionEU" at the end of the structure. This will include any info that occurs in all regions, but may be region specific such as the SDS version number. (Later, other regions will be added in the same way.)

**Comment by dirk-qualisys** [comment]
While in most sections, "RegionEU" will cover small parts of the data, this is different in section 2: Classification is region specific, the labelling is derived from the classification, and even section 2.3 "Other hazards" is different in different regions since US hazard classes cover topics that EU data sheets would cover in 2.3.
First checkin covers sections 1 to 11. Others will follow.

**Comment by dirk-qualisys** [comment]
Further impacts of internationalization:

- Composition is region specific: Due to differingcut-off limits on CMR classification, legal classification lists, SCLs in Europe etc., not nly the product classification, but also the list of ingredients may differ (and thus needs to be repeated for every region).
- Transport info needs to be structured by carriers: Just like in the good old days of pre-REACH SDSs! Since region-specific transport info exists, it makes most sense to apply this to IMDG and IATA-DGR as well. It is only Europe that deviates from this structure in printout.

**Comment by dirk-qualisys** [comment]
There is a lot to say about internationalization of classification and labelling, so I will split this into a separate issue...
